### PR TITLE
Bump: Sentry Android to 5.5.1 and Sentry Cocoa to 7.7.0

### PR DESCRIPTION
**Pull request by marandaneto**
## :loudspeaker: Type of change

- [ ] Bugfix
- [ ] New feature
- [X] Enhancement
- [ ] Refactoring

## :scroll: Description
Bump: Sentry Android to 5.5.0 and Sentry Cocoa to 7.6.1

## :pencil: Checklist

- [X] I reviewed submitted code
- [ ] I added tests to verify changes
- [X] All tests passing
- [X] No breaking changes

### CODE REVIEW

**Comment by btrigueiro** [comment]
thank you @marandaneto!
had been looking forward for this update with `sentry-android@5.5.0`

**Comment by btrigueiro** [comment]
iOS E2E failure seems like it could be sorted with a re run of CI
https://github.com/getsentry/sentry-react-native/runs/4522577312?check_suite_focus=true#step:17:25

**Comment by marandaneto** [comment]
@jennmueng ideas why the iOS e2e tests are failing?

**Comment by btrigueiro** [comment]
@jennmueng @romtsn could you please re trigger the action?
or maybe @marandaneto can push some change so CI runs again?

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Unreleased
+
+- Bump: Sentry Android to 5.5.0 and Sentry Cocoa to 7.6.1 #1965
```
Comment (marandaneto) [inline CHANGELOG.md:5]:
```suggestion
- Bump: Sentry Android to 5.5.0 and Sentry Cocoa to 7.7.0 #1965
```

File: RNSentry.podspec [hunk 2]
```diff
@@ -17,7 +17,7 @@ Pod::Spec.new do |s|
   s.preserve_paths = '*.js'
 
   s.dependency 'React-Core'
-  s.dependency 'Sentry', '7.5.4'
+  s.dependency 'Sentry', '7.6.1'
```
Comment (marandaneto) [inline RNSentry.podspec:20]:
```suggestion
  s.dependency 'Sentry', '7.7.0'
```

**Review by jennmueng — APPROVED** [review]
LGTM but I'm trying to fix ios e2e tests rn

File: CHANGELOG.md [hunk 3]
```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## Unreleased
+
+- Bump: Sentry Android to 5.5.0 and Sentry Cocoa to 7.7.0 #1965
```
Comment (marandaneto) [inline CHANGELOG.md:5]:
```suggestion
- Bump: Sentry Android to 5.5.1 and Sentry Cocoa to 7.7.0 #1965
```

File: android/build.gradle [hunk 4]
```diff
@@ -24,5 +24,5 @@ android {
 
 dependencies {
     implementation 'com.facebook.react:react-native:+'
-    api 'io.sentry:sentry-android:5.4.3'
+    api 'io.sentry:sentry-android:5.5.0'
```
Comment (marandaneto) [inline android/build.gradle:27]:
```suggestion
    api 'io.sentry:sentry-android:5.5.1'
```

**Pull request merged**